ledger/src/blocktree_db.rs

```diff
@@ -580,8 +580,8 @@ impl Database {
         self.backend.write(batch.write_batch)
     }
 
-    pub fn storage_size(&self) -> u64 {
-        get_size(&self.path).expect("failure while reading ledger directory size")
+    pub fn storage_size(&self) -> Option<u64> {
```

Comment:
Might be nice to have a comment here as to why it's an option. Feel free to do it in a followup

Reply:
Seems like returning the Result would be self-documenting

Reply:
oops didn't see this, will touch the code up along these lines